src/contentScript/sidebarController.tsx

```diff
@@ -151,15 +150,57 @@ export function showSidebar(callbacks = extensionCallbacks): string {
     }
   }
 
+  if (!isEmpty(activateOptions)) {
```

Comment:
Why `activateOptions` can't be `null` by default? Then the check will be simpler: 
```suggestion
  if (activateOptions != null) {
```

Reply:
> Why activateOptions can't be null by default? Then the check will be simpler

I think keeping track of null vs. non-null is generally more error-prone. If there's a valid "empty"/default value I like to use that instead